doc/installation.rst

```diff
@@ -11,10 +11,10 @@ Thanks!
 
 .. _issue tracker: https://github.com/spyder-ide/spyder/issues
 
-.. important::
+.. note::
 
-   Spyder now offers :ref:`standalone installers<standalone_installers_ref>` for Windows and macOS, making it easier to get up and running with the application without having to download Anaconda or manually install it in your existing environment.
-   While we still support Anaconda, we recommend this install method on those platforms to avoid most problems with package conflicts and other issues.
+   For most users on Windows and macOS, we recommend using our :ref:`standalone_installers_ref`, and for those on Linux or who need third-party Spyder plugins, which the standalone installers currently do not support, we advise using :ref:`installation-conda`.
```

Comment:
> Say why we recommend our installers.

I actually included such in several previous drafts, but I didn't want to duplicate the information in the standalone installers section, so I ended up eliding it in the end. However, I revised it to mention this again (with more appropriate diction).

> Mention other packages because that's important for users.

The specific limitation here with the standalone installers and third party packages is the ability to view the latter's data types in the Variable Explorer (assuming a solution to that hasn't been implemented yet), which I added a mention of.

Otherwise, for non-included packages, users just have to use a separate Python environment to run their code (provided by an existing or new Anaconda/Miniconda/etc installation, in this case), which is what we've always told them to do anyway for years and years now to avoid breaking Spyder, and is the whole point of the standalone installers. This is described in more detail, with a link to our FAQ and How-To video on the subject, at the top of the standalone installers section, and I also include more guidance on setting this up in the Conda section.